docs/api/client/hoodie.account.rst

```diff
@@ -26,23 +26,15 @@ Example
 
     hoodie.account.on('signout', redirectToHome)
 
-
-.. _label-hoodie-account-ready:
-
-hoodie.account.ready
---------------------
-
-`Read-only`. Promise that resolves once the account instance loaded its current state from the store.
-
 hoodie.account.id
 -----------------
 
-`Read-only`. Returns the account id. Cannot be accessed until the :ref:`label-hoodie-account-ready` promise resolved.
+`Read-only`. Returns the account id.
 
 hoodie.account.username
 -----------------------
 
-`Read-only`. Returns the username if signed in, otherwise ``undefined``. Cannot be accessed until the :ref:`label-hoodie-account-ready` promise resolved.
+`Read-only`. Returns the username if signed in, otherwise ``undefined``.
```

Comment:
we also removed `.id` and `.username`. You can compare to https://github.com/hoodiehq/hoodie-account-client. You can remove them as part of your PR or make a follow up one, as you prefer :)